### DISCUSSION TITLE
Node / node-sass version errors

### DISCUSSION THREAD

**Discussion by Makahdm** [pxblue/react-component-library#618] [Q&A] [answered]
could you tell me the node version and node-sass version?
i failed to comple it at the step of  “yarn start:showcase”
thank u so much!!!!

**Comment by joebochill** [comment, marked as answer]
The node-sass errors you are seeing are coming from the demos/storybook project. The `start:showcase` command has a sub-step that installs dependencies for all of the demo projects and that's where the error comes from. We're in the process of removing the storybook demo and replacing it with something much better (to be released in the next couple days).

In the meantime, you should be able to run the showcase by doing the following instead of `yarn start:showcase`:

```sh
# clone the repo
yarn initialize # pulls in the showcase demo submodule
cd demos/showcase
yarn start
```

If you just want to run the showcase and aren't concerned with testing out components from the component library that are in development, you can also run the showcase from its dedicated repo (https://github.com/brightlayer-ui/react-showcase-demo) instead of as a submodule of the component library repo.
